src/tests/ftest/util/ddb_utils.py

```diff
@@ -60,13 +61,11 @@ def run(self):
         """Run the command.
 
         Returns:
-            list: A list of dictionaries with each entry containing output, exit status,
-                and interrupted status common to each group of hosts.
+            CommandResult: groups of command results from the same hosts with the same return status
 
         """
-        return run_pcmd(
-            hosts=self.host, command=str(self), verbose=self.verbose,
-            timeout=self.timeout)
+        return run_remote(
```

Comment:
It looks like this will break `recovery/ddb.py` and `recovery/container_list_consolidation.py` as they do things like:
```
            cmd_result = ddb_command.list_component()
            ls_out = "\n".join(cmd_result[0]["stdout"])

Reply:
They are already updated

Reply:
Ah, sorry I missed the update.